### ISSUE TITLE
plot(section) fails if the field has only NA values

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1007]
This came up while I was working on #1006 (because the problems were in a funky dataset that led to 1006). Below is a test code that doesn't use the funky dataset:
```R
library(oce)
data(section)
for(i in seq_along(section@data$station)) {
    section@data$station[[i]]@data$salinity <- rep(NA,
          length(section@data$station[[i]]@data$salinity))
}
plot(section,which='salinity')
```

**Comment by dankelley** [comment]
This is ok now in `kelley` but I'll leave this open until I merge into `develop`, likely on June 30

**Comment by dankelley** [comment]
Oh, and also I want the `image` scheme to draw blank axis frames, as the `contour` scheme does. (For that see #1006 test code)

**Comment by dankelley** [comment]
Fixed in `develop` commit 55b523eb63fb97767091cde1686a45d619d0d531
